docs/concepts/updating.md

```diff
@@ -23,6 +23,7 @@ below.
 | 1.0.0-1.0.2   | >=10.8.1 <11.0.0  | >=5.2.0 <6.0.0          | >= 1.23.0 |
 | 1.1.0-1.1.5   | >=10.8.1 <11.0.0  | >=5.2.0 <6.0.0          | >= 1.25.0 |
 | 1.2.0         | >=10.15.0 <11.0.0 | >=6.1.0                 | >= 1.25.0 |
+| 1.2.1         | >=10.15.0 <11.0.0 | >=6.1.0                 | >= 1.35.0 |
```

Comment:
Super tiny nit: We have some unreleased features in `main`, so the next version will be very likely `1.3.0`.

```suggestion
| 1.3.0         | >=10.15.0 <11.0.0 | >=6.1.0                 | >= 1.35.0 |
```